app/views/conversions/academy_urn/confirm.html.erb

```diff
@@ -0,0 +1,39 @@
+<%= render partial: "projects/shared/signed_in_as", locals: {email_address: current_user.email} %>
```

Comment:
We should really just delete this and stop propagating it!! Not something you should do here though!